invenio_app_ils/config.py

```diff
@@ -1009,6 +939,58 @@ def _(x):
             order=2,
         ),
     ),
+    acq_orders=dict(  # OrderSearch.Meta.index
```

Comment:
Do you think we should move inside the module `config.py`?

Reply:
The reasoning behind putting it in ILS `config.py` was that these sorting options are based on our needs so if it were to become its own module it would make more sense to have it in ILS. However, I don't see any harm in providing these sort options in acquisition `config.py` as an example for others. Do you have any preference?